### PR TITLE
🚀 RELEASE 3.1.3:  Update release notes

### PR DESCRIPTION
**Pull request by MrCordeiro**
- Update release notes
- Update README.md
- Update CONTRIBUTORS.md

### CODE REVIEW

**Comment by MrCordeiro** [comment]
@ArnarTumi , @jnpkr, pls check if that's how you prefer your names to be displayed.

File: docs/source/releases/3.1.3.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+===============================================
+Wagtailmenus 3.1.2 release notes (15.10.2022)
```
Comment (MrCordeiro) [inline docs/source/releases/3.1.3.rst:2]:
```suggestion
Wagtailmenus 3.1.3 release notes (15.10.2022)
```

Comment (MrCordeiro) [reply docs/source/releases/3.1.3.rst:2]: Ops, wrong version. (hooray for double checking!)

**Comment by ArnarTumi** [comment]
> @ArnarTumi , @jnpkr, pls check if that's how you prefer your names to be displayed.

Looks good!

**Comment by MrCordeiro** [comment]
Ok, we're live! Time for some 🍻!

**Pull request merged**